### DISCUSSION TITLE
how to merge similar legends in status timeline

### DISCUSSION THREAD

**Discussion by girishmanyam** [grafana/grafana#53940] [Dashboards, Panels & Visualizations]
Hello,

I am plotting a graph using status timeline against 4 metrics eg waiting, arrived, arrived_late and over due. Metrix/legend - waiting is appearing in 2 lines instead of one.  How to merge the legends waiting( or metric ) as shown in attached pic.?

thanks in advance

<img width="749" alt="grafanapic" src="https://user-images.githubusercontent.com/2779562/185607354-5106ab2e-d43a-4a64-a04b-d5d3fe4bfcdc.PNG">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
